pkg/agent/datamodel/types.go

```diff
@@ -2106,3 +2107,22 @@ func (a *AgentPoolProfile) IsDisableWindowsOutboundNat() bool {
 		a.AgentPoolWindowsProfile.DisableOutboundNat != nil &&
 		*a.AgentPoolWindowsProfile.DisableOutboundNat
 }
+
+// SecurityProfile begin.
+type SecurityProfile struct {
+	PrivateEgress *PrivateEgress `json:"privateEgress,omitempty"`
+}
+
+type PrivateEgress struct {
+	Enabled      bool   `json:"enabled"`
```

Comment:
Add `ContainerRegistryFQDN string` so that we can use it in `getOutBoundCmd`.
https://github.com/Azure/AgentBaker/blob/master/pkg/agent/variables.go#L173

Reply:
added ContainerRegistryServer